addons/web/static/src/setup.js

```diff
@@ -23,6 +23,7 @@ export async function startWebClient (Webclient) {
     server_version: sessionInfo.server_version,
     server_version_info: sessionInfo.server_version_info,
   };
+  odoo.isReady = false;
```

Comment:
will need to reexplain to me what this does

Reply:
it is used by the click_everywhere test to determine when it can start clicking everywhere. We could probably find another way, but I thought that the flag was legitimate